### DISCUSSION TITLE
Support for Meross MSS315 power plug?

### DISCUSSION THREAD

**Discussion by slackstarr** [krahabb/meross_lan#292] [General]
Hi, 

Do you plan to support the just released Meross Smart Power Plug MSS315?

https://shop.meross.com/products/meross-matter-plug-with-energy-monitor-mss315-eu MSS

Many thanks!
/Marco

**Comment by slackstarr** [comment]
Please ignore my post. Device just showed up. Sorry for not being more patient. ;-)

**Reply by krahabb** [reply]
No worries...
At any rate, I've seen those plugs are also 'Matter' enabled so they should also work with the native HA Matter integration
